fedmsg_meta_fedora_infrastructure/tests/greenwave.py

```diff
@@ -52,7 +52,7 @@ class TestGreenwaveDecisionChange(Base):
         "crypto": "x509",
         "topic": "org.fedoraproject.prod.greenwave.decision.update",
         "msg": {
-            "policies_satisified": True,
+            "policies_satisfied": True,
```

Comment:
We'll need to keep the previous test cases and just add new tests for the fixed versions.

Reply:
ACK! So I'll rebase with only the other change. Thanks